--- a/cupsfilters/colord.h
+++ b/cupsfilters/colord.h
@@ -35,6 +35,7 @@
 /* Common routines for accessing the colord CMS framework */
 
 #include <cups/raster.h>
+#include <cups/ppd.h>
 
 char  **colord_get_qualifier_for_ppd      (ppd_file_t *ppd);
 char   *colord_get_profile_for_device_id  (const char *device_id,

--- a/cupsfilters/driver.h
+++ b/cupsfilters/driver.h
@@ -38,6 +38,7 @@

 #  include <cups/cups.h>
 #  include <cups/raster.h>
+#  include <cups/ppd.h>


 /*

